settings.yaml

```diff
@@ -34,7 +34,7 @@ urlHost: "serial-vault"
 urlScheme: https
 
 # Enable user authentication using Ubuntu SSO
-enableUserAuth: True
+#enableUserAuth: True
```

Comment:
commented out on purpose? why?

Reply:
Just done for testing. I'll change it back.